packages/connect/src/data/firmwareInfo.ts

```
@@ -17,14 +17,22 @@
 
 // undefined releases should never happen for official firmware, only custom
 const releases = Object.values(DeviceModelInternal).reduce(
     (acc, key) => ({ ...acc, [key]: [] }),
     {} as Record<keyof typeof DeviceModelInternal, FirmwareRelease[] | undefined>,
 );
 
-export const parseFirmware = (json: any, deviceModel: DeviceModelInternal) => {
+// `releasesRemoteLatest` is collecting the releases.json from data.trezor.io
+// we keep the original `releases that was bundled.
+const releasesRemoteLatest = Object.values(DeviceModelInternal).reduce(
+    (acc, key) => ({ ...acc, [key]: [] }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)